kbase-extension/static/kbase/js/widgets/vis/kbaseTreechart.js

```diff
@@ -2,354 +2,342 @@
 
 */
 
-define (
-	[
-		'kbwidget',
-		'bootstrap',
-		'jquery',
-		'd3',
-		'kbaseVisWidget',
-		'RGBColor',
-		'geometry_rectangle',
-		'geometry_point',
-		'geometry_size'
-	], (
-		KBWidget,
-		bootstrap,
-		$,
-		d3,
-		kbaseVisWidget,
-		RGBColor,
-		geometry_rectangle,
-		geometry_point,
-		geometry_size
-	) => {
-
+define([
+    'kbwidget',
+    'bootstrap',
+    'jquery',
```

Comment:
Lots of AST changes in this file, but they appear not to have an effect on the output. Git line numbers are messed up, so the diff is a lot larger than it would otherwise be.